addons/mail/static/src/components/dialog/dialog.xml

```diff
@@ -4,17 +4,12 @@
     <t t-name="mail.Dialog" owl="1">
         <div class="o_Dialog">
             <t t-if="dialog">
-                <t t-if="dialog.record">
```

Comment:
not sure whether it's safe to remove this guard

Reply:
As discussed, `record` now ensured required on `dialog` so guard on `dialog` is enough.